_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,15 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added readonly attribute to aria-disabled.
+    summaryAdditional: Now, the input element and calendar icon remain discoverable but disabled behaviors are replicated.
```

Comment:
**suggestion:** Since the change covers more than just adding the `readonly` attribute we could maybe reword to something along the lines of:

```suggestion
    summary: Enhanced `aria-disabled` behaviors to match `disabled` behaviors.
    summaryAdditional: Now, the input element and calendar icon remain discoverable to screen readers but cannot be interacted with.
```

Feel free to toy with it!

Reply:
Resolved in [fefb694](https://github.com/uswds/uswds-site/pull/2794/commits/fefb6949907954544486e519967cf3c9c5b7f530)